src/assets/styles/modules/youtube.module.scss

```diff
@@ -0,0 +1,5 @@
+.youtube{
+    visibility: hidden;
+    display: none;
+    width: 0px;
```

Comment:
This can be shortened to just 0 without the unit @moria0525

Reply:
thank. I fixed